Makefile

```diff
@@ -0,0 +1,76 @@
+# Copyright 2023 LiteFarm.org
+# This file is part of LiteFarm.
+#
+# LiteFarm is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# LiteFarm is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+
+include .env.secrets
```

Comment:
I had to make this an optional include (`-include`) to prevent crashing on the first run with this error. After that change it worked!

<img width="624" alt="Screenshot 2023-08-29 at 3 44 44 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/3de8f57a-7fd8-44e0-9ae7-6fb21eefa928">